src/include/simeng/Core.hh

```diff
@@ -13,6 +14,40 @@ class ArchitecturalRegisterFileSet;
 
 enum CoreStatus : uint8_t { halted, idle, executing, switching };
 
+/** The result from a handled exception. */
+struct ExceptionResult {
```

Comment:
space between all variables in the this struct

Reply:
please add `_` to all member variables

Reply:
As far as I remember, we don't tend to add `_` to the end of struct variables

Reply:
I thought the guideline was to add `_` to every member variable? In the virtmem PR all structs have `_`. Given that this isn't super important rn. I'll resolve it.